client/src/App.js

```diff
@@ -22,8 +23,10 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route path="/" exact component={MovieList}/>
+        <Route path="/movies/:id" component={Movie}/>
```

Comment:
great job only adding the exact param to the '/' route since the other one does not need it in this application